pyccel/codegen/printing/cwrappercode.py

```diff
@@ -136,4 +141,4 @@ def cwrappercode(expr, assign_to=None, **settings):
         ``(*a)`` instead of ``a``.
     """
 
-    return CWrapperCodePrinter(settings).doprint(expr, assign_to)
```

Comment:
The CWrapperCodePrinter should print a module not a Codegen object

Reply:
Yes I was going to correct this one I was  little confused on how  this class work